hls4ml/converters/keras/core.py

```diff
@@ -103,7 +103,7 @@ def parse_activation_layer(keras_layer, input_names, input_shapes, data_reader,
 
 @keras_handler('BatchNormalization')
 def parse_batchnorm_layer(keras_layer, input_names, input_shapes, data_reader, config):
-    assert('BatchNormalization' in keras_layer['class_name'])
+    assert('BatchNormalization' in keras_layer['class_name'] or 'QConv2DBatchnorm' in keras_layer['class_name'])
```

Comment:
I think this is okay for the time being. In the end, it would be nice if the `BatchNormalization` layer converter does not need to know about the extra layer. For example, when we get `QDenseBatchnorm` as well it would need to go here. But the best solution for this would be if QKeras just renames the layers, so let's leave it like this for now and wait.